runtime/layers/fpm/php-fpm.conf

```diff
@@ -25,3 +25,6 @@ decorate_workers_output = no
 ; Limit the number of core dump logs to 1 to avoid filling up the /tmp disk
 ; See https://github.com/brefphp/bref/issues/275
 rlimit_core = 1
+
+; Make sure no PHP request takes longer than 27s
+request_terminate_timeout = 27s
```

Comment:
For some reason this doesn't seem relevant to me. I'm using ALB, so I'm not restricted by 30 seconds and if I increase the number of ReadWriteTimeout (https://github.com/brefphp/bref/blob/master/src/Event/Http/FpmHandler.php#L81), I can get an API to do `sleep(50)` to work just fine with FPM.

Reply:
See #708

Reply:
My point is that to me it seems that this variable could be any number and it wouldn't matter at all. I can still have requests running for more than 27 seconds even though this indicates that I technically shouldn't?